actions/profile_db.py

```diff
@@ -80,7 +81,7 @@ class RecipientRelationship(Base):
     recipient_nickname = Column(String(255))
 
 
-def create_database(database_engine, database_name):
+def create_database(database_engine: sa.engine.base.Engine, database_name: Text):
```

Comment:
This works, we could also import the class directly as we do for other sa classes and just reference it as `Engine` -- up to you. also thank you for adding classes 🙌🏻

Reply:
Ah right, will do.